redaxo/src/core/lib/console/package/activate.php

```diff
@@ -15,7 +15,19 @@ protected function configure()
     {
         $this
             ->setDescription('Activates the selected package')
-            ->addArgument('package-id', InputArgument::REQUIRED, 'The id of the package (addon or plugin); e.g. "cronjob" or "structure/content"');
+            ->addArgument('package-id', InputArgument::REQUIRED, 'The id of the package (addon or plugin); e.g. "cronjob" or "structure/content"', null, static function() {
+                $packageNames = [];
+
+                foreach(rex_package::getRegisteredPackages() as $package) {
+                    if ($package->isAvailable()) {
+                        continue;
+                    }
+
+                    $packageNames[] = $package->getPackageId();
+                }
+
+                return $packageNames;
```

Comment:
Man könnte es auch so notieren:

```suggestion
                return array_keys(array_filter(
                    rex_package::getRegisteredPackages(), 
                    fn (rex_package $package) => !$package->isAvailable(),
                ));
```

Kürzer. Aber keine Ahnung, ob besser lesbar.

Reply:
ich bin kein fan von `array_filter` da mand en code dann "rückwärts" lesen muss. die einfache schleife würde ich bevorzugen